File: benchmark_results/Python/两数之和/test_solution.py
from solution import two_sum

def test_case_1():
    assert sorted(two_sum([2, 7, 11, 15], 9)) == [0, 1]

def test_case_2():
    assert sorted(two_sum([3, 2, 4], 6)) == [1, 2]

def test_case_3():
    assert sorted(two_sum([3, 3], 6)) == [0, 1]

def test_case_4():
    assert sorted(two_sum([0, 4, 3, 0], 0)) == [0, 3]

def test_case_5():
    assert sorted(two_sum([], 0)) == []

File: benchmark_results/Java/两数之和/TestSolution.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class TestSolution {
    @Test
    void testCase1() {
        Solution solution = new Solution();
        int[] result = solution.twoSum(new int[]{2, 7, 11, 15}, 9);
        assertArrayEquals(new int[]{0, 1}, result);
    }

    @Test
    void testCase2() {
        Solution solution = new Solution();
        int[] result = solution.twoSum(new int[]{3, 2, 4}, 6);
        assertArrayEquals(new int[]{1, 2}, result);
    }

    @Test
    void testCase3() {
        Solution solution = new Solution();
        int[] result = solution.twoSum(new int[]{3, 3}, 6);
        assertArrayEquals(new int[]{0, 1}, result);
    }

    @Test
    void testCase4() {
        Solution solution = new Solution();
        assertThrows(IllegalArgumentException.class, () -> {
            solution.twoSum(new int[]{1, 2, 3}, 7);
        });
    }
}